examples/gcp_gpu_direct_tcpx/tcpx_sglang_serving.yaml

```diff
@@ -0,0 +1,50 @@
+name: tcpx-sglang-serving
+
+resources:
+  cloud: gcp
+  instance_type: a3-highgpu-8g
+  network_tier: best
+  disk_size: 1024
+  disk_tier: best
+  ports: 30000
+
+num_nodes: 2 # Specify number of nodes to launch; requirements may vary based on accelerators
+
+setup: |
+  python3 -m pip install uv
```

Comment:
During SkyPilot runtime setup, `uv` should be automatically setup already.

Reply:
Removed